changelogs/fragments/bugfix_vrf_range_resolution.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - "acls - fix lookup of range port conversion from int to string to allow strings. (https://github.com/ansible-collections/cisco.nxos/issues/???)."
```

Comment:
Make sure we update the issue link

Reply:
@KB-perByte I've updated this to point to redhat, or should we point somewhere else